jakelib/deploy.jake

```diff
@@ -55,7 +62,7 @@ const getAnalyticsCommands = (subName) => {
 
 const apps = {
   // CORE API
-  consoleapi: { cmds: coreRootCommands },
+  'consoleapi': { cmds: coreRootCommands },
```

Comment:
do we want a hyphen here to standardize it?

Reply:
That will take so much work in re-terraforming services.  I thought about it but this is the existing value and didn't want to update it.

Reply:
np, we did it in our branch. just wondering if you had too